BEProjectsApp/models.py

```diff
@@ -1,3 +1,81 @@
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+
+DOMAIN_CHOICES = [
+    (1, ("none")),
```

Comment:
You can remove this.

Reply:
why?

Reply:
It has no need as we need only the required domains list.

Reply:
Oh you mean just the 1st one, I mean the 'none' choice ?

Reply:
Yes, as we can't give people **none** as a choice.